.github/workflows/integration-test.yml

```diff
@@ -7,13 +7,12 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        tutor_version: ["<17.0.0", "<18.0.0", "<19.0.0"]
+        tutor_version: ['<18.0.0', '<19.0.0', 'nightly']
     steps:
-      - uses: actions/checkout@v4
-        with:
-          path: eox-tenant
-      - uses: eduNEXT/integration-test-in-tutor@main
+      - name: Run Integration Tests
+        uses: eduNEXT/integration-test-in-tutor@mjh/make-extra-reqs-step-optional
         with:
           tutor_version: ${{ matrix.tutor_version }}
-          app_name: "eox-tenant"
-          shell_file_to_run: "eox_tenant/test/tutor/integration.sh"
+          app_name: 'eox-tenant'
+          shell_file_to_run: 'scripts/execute_integration_tests.sh'
```

Comment:
This input is necessary? since there is already a default with the same path in the action.

Reply:
@BryanttV you are right, I forgot we had a default for this, thanks for noticing!